public/doc/openapi-3-v1.0.0.json

```diff
@@ -1296,6 +1296,12 @@
                     "description": {
                         "type": "string",
                         "readOnly": true
+                    },
+                    "created_at": {
```

Comment:
Although adding this field does not hurt, why it is mandatory?

Reply:
Because in spec file, we expect `created_at` in `iso.8601` format. We have to add here so the out resource has this attribute.

Reply:
we should change spec test file rather than api spec.